spec/site_prism/element_checker_spec.rb

```diff
@@ -105,6 +105,19 @@
       end
     end
 
+    describe '#missing_elements' do
+      let(:present) { expected_items[2..-1] }
+      let(:missing) { expected_items[0..1] }
+
+      it 'returns missing elements' do
+        present.each { |name| expect(page).to receive(:there?).with(name).once.and_call_original }
```

Comment:
At the moment this won't bork rubocop, but down the line I plan on having 1 spec per it. It's one of the tasks I've got to clean up.

Do you think we "should" have 1 spec per it, and if so, can we refactor this slightly maybe into more tests?

Reply:
I don't see the good way to split test to several its. Potentially replace expect with allow, but in this case, it won't verify it is called for each expected_item. Any suggestions?
I'm usually ok having multiple expects in it if they are closely related to each other.

Reply:
I don't know either. Happy for you to maybe offer some advice here.

Maybe we could change the logic in the repo to allow 2 assertions. But I would like to try keep as close to rubocop rspec standards as is possible. Could we maybe have multiple tests?

I'm not really sure on this one.